smartsim/_core/mli/infrastructure/worker/worker.py

```diff
@@ -71,11 +74,15 @@ def __init__(
         outputs: t.Optional[t.Collection[t.Any]] = None,
         output_keys: t.Optional[t.Collection[str]] = None,
         failed: bool = False,
+        status_enum: "StatusEnum" = "complete",
```

Comment:
Should probably _use_ the enum if we declare the type as such? e.g.

`status: StatusEnum = StatusEnum.complete`

Reply:
It's a `StatusEnum` on the `capnproto` schema side of things, so we need to set it to be one of its accepted strings. `StatusEnum.complete` doesn't do the trick but I really wish it did.

from `pycapnp` quickstart guide:
```
struct PhoneNumber {
    number @0 :Text;
    type @1 :Type;

    enum Type {
      mobile @0;
      home @1;
      work @2;
    }
  }


Enums are treated like strings, and you assign to them like they were a Text field:

alicePhone.type = 'mobile'
```

Reply:
Understood. Any issues with avoiding `XxxEnum` naming and just do `Status` as the field name here?

Reply:
`StatusEnum` -> `Status` is complete!